### DISCUSSION TITLE
No group information in group license file

### DISCUSSION THREAD

**Discussion by SajmalBaheesh** [dashingsoft/pyarmor#1267] [Q&A]
Hi,
I have a group license and trying to do the offline installation. I followed the steps given in the document, generated the device file then copied to the initial build machine, generated the zip file, and copied to the offline machine.
Getting error: no group information in group license file** 
                      Invalid group license file "pyarmor-device-regfile-6018.4.zip"

I am trying to setup pyarmor 8 in ubuntu 22.04. This system has Python 3.10 and 3.8. My Python script is for Python 3.8 and the pyarmor is correctly detecting this version.
My initial build machine is ubuntu 20.4

**Comment by jondy** [comment]
Please upgrade pyarmor to 8.2.1 and also upgrade `pyarmor.cli.core` to 3.2.1.

And use `pyarmor -d reg ...` to print more information in the console to check what happens.
